city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    def parse(self, response):
+        """   `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping needs.   """
+        meeting_location = ''
+
+        for item in response.css("#content-232764 div.panel-body p"):
+
+            if item.css("p > strong ::text").getall():
```

Comment:
Once you start checking committees, it might be easier to pass the `response` to the function checking the location since the location won't be in the second block of items. We've been storing any location that we're validating in `self.location` and calling the method `_validate_location` in this case just to make things more clear. You can see an example here https://github.com/City-Bureau/city-scrapers/blob/c23f1fd17b1ced1c01dff295c8cdf1ca191f0244/city_scrapers/spiders/chi_ssa_34.py#L28

Reply:
I'll look at this a little later.